evap/staff/views.py

```diff
@@ -2179,6 +2178,21 @@ def course_type_merge(request, main_type_id, other_type_id):
     )
 
 
+@manager_required
+class ExamTypeIndexView(SuccessMessageMixin, SaveValidFormMixin, FormsetView):
+    model = ExamType
+    formset_class = modelformset_factory(
+        ExamType,
+        form=ExamTypeForm,
+        formset=ModelWithImportNamesFormset,
+        can_delete=True,
+        extra=1,
+    )
+    template_name = "staff_exam_type_index.html"
+    success_url = reverse_lazy("staff:exam_type_index")
```

Comment:
Why reverse_lazy? (If there is a reason, maybe add a comment?)

Reply:
Isn't this always used for class based views because the urls module imports the views ? In that case I wouldn't comment